tests/mock_logger.h

```diff
@@ -32,7 +36,32 @@ class MockLogger : public multipass::logging::Logger
     MockLogger() = default;
     MOCK_CONST_METHOD3(log, void(multipass::logging::Level level, multipass::logging::CString category,
                                  multipass::logging::CString message));
+
+    template <typename Matcher>
+    static auto make_cstring_matcher(const Matcher& matcher)
+    {
+        return Property(&multipass::logging::CString::c_str, matcher);
+    }
```

Comment:
There's another one of those here:
https://github.com/canonical/multipass/blob/dfaa8492602c96a099d98225baec0845fe0d1f0c/tests/mock_logger.h#L40-L44

Reply:
Good catch. Fixed.